### PR TITLE
Documentation for Sugarizer IOS build

### PR DESCRIPTION
**Pull request by sarthak-g**

### CODE REVIEW

File: docs/ios/ios_doc.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+![buildsettings](https://user-images.githubusercontent.com/42293606/121804462-26855400-cc64-11eb-9727-baabe627e19e.JPG)
+
+Scroll down you'll find "Swift Compiler - Language" section. In this section, select "Swift Language Version" as 3 or above
+
+![Swift Language Setting](https://user-images.githubusercontent.com/42293606/121804549-8976eb00-cc64-11eb-9f33-fe107a7a3ecd.JPG)
+
+Then go to "Sugarizer/plugins/" directory and open "CDVFileTransfer.m" file. Comment line 107-110 and save the file.
+
+![CDVFileTransfer comment](https://user-images.githubusercontent.com/42293606/121804587-bc20e380-cc64-11eb-9843-ffc6e2a9624e.JPG)
+
```
Comment (llaske) [inline docs/ios/ios_doc.md:37]: It's not enough. As mentioned [here](https://github.com/llaske/sugarizer/blob/dev/config.xml#L80), a patched version of the `cordova-plugin-wkwebview-file-xhr` is need on iOS.
So a git clone of [this repo](https://github.com/llaske/cordova-plugin-wkwebview-file-xhr) is require to generate the file.

File: docs/ios/make_ios.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+echo --- Copying content
+rsync -av --exclude-from='exclude.ios' ../sugarizer/* www
+cp ../sugarizer/config.xml .
+sed -i -e "s/\({\"id\": \"org.sugarlabs.TurtleBlocksJS\",.*\},\)//" www/activities.json
+sed -i -e "s/\({\"id\": \"org.somosazucar.JappyActivity\",.*\},\)//" www/activities.json
+rm -rf www/activities/Jappy.activity www/activities/TurtleBlocksJS.activity
+
+cordova platform add ios
+cordova plugin add @ahovakimyan/cordova-plugin-wkwebviewxhrfix
```
Comment (llaske) [inline docs/ios/make_ios.sh:27]: Should be replaced by my version. See [here](https://github.com/llaske/cordova-plugin-wkwebview-file-xhr#about-sugarizer-update) why.

**Review by llaske — CHANGES_REQUESTED** [review]
Nice. But few more things to update.
See my comments.

**Comment by sarthak-g** [comment]
@llaske , made suggested changes. Please review.

**Comment by llaske** [comment]
Ok thanks.

**Pull request merged**